### PR TITLE
Add Bernoulli likelihood

### PR DESCRIPTION
**Pull request by sharanry**
Adds Bernoulli likelihood which would enable us to easily model binary classification tasks.

### CODE REVIEW

File: src/likelihoods/bernoulli.jl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+```math
+    p(y|f) = Bernoulli(y | f)
+```
+On calling, this would return a Bernoulli distribution with `f` probability of `true`.
+"""
+struct BernoulliLikelihood end
+
+@functor BernoulliLikelihood
+
+_logistic(logit::Real) = 1/(1 + exp(-logit))
```
Comment (theogf) [inline src/likelihoods/bernoulli.jl:16]: You can do like in #14 and use `logistic` from `StatsFuns`

**Comment by sharanry** [comment]
Is it okay to merge this?

File: src/likelihoods/bernoulli.jl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+uncertainity associated with the data follows a Bernoulli distribution.
+
+```math
+    p(y|f) = Bernoulli(y | f)
+```
+On calling, this would return a Bernoulli distribution with `f` probability of `true`.
+"""
+struct BernoulliLikelihood end
+
+@functor BernoulliLikelihood
```
Comment (devmotion) [inline src/likelihoods/bernoulli.jl:14]:
```suggestion
```

**Comment by devmotion** [comment]
Same here, I'm not sure if @willtebbutt's intention was that https://github.com/JuliaGaussianProcesses/GPLikelihoods.jl/issues/18 should be addressed first.

**Comment by willtebbutt** [comment]
Same here. Feel free to merge and refactor one test utils are sorted.

**Pull request merged**